src/e-pr.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+const childProcess = require('child_process');
+const open = require('open');
+const path = require('path');
+const program = require('commander');
+
+const evmConfig = require('./evm-config');
+const { fatal } = require('./util');
+
+function guessPRTarget(config) {
+  const package = require(path.resolve(config.root, 'src', 'electron', 'package.json'));
+  if (package.version.includes('nightly')) {
+    return 'master';
+  }
+  const match = /^([0-9]+)\.([0-9]+)\.[0-9]+.*$/.exec(package.version);
+  if (match) {
+    return `${match[1]}-${match[2]}-x`;
+  }
+  throw `Failed to determine target PR branch`;
```

Comment:
Same as above - can we make this more actionable? This failing bc it didn't match the regex, so we should tell the users that

Reply:
Oops, I fixed this before seeing this comment, so it's addressed in a force-push. :roll_eyes: